core/src/main/scala/kafka/tools/TestRaftServer.scala

```diff
@@ -160,7 +160,7 @@ class TestRaftServer(
       eventQueue.offer(HandleClaim(epoch))
     }
 
-    override def handleResign(): Unit = {
+    override def handleResign(epoch: Int): Unit = {
```

Comment:
can use "_" since we don't care about the value

Reply:
> can use "_" since we don't care about the value

I can't get it to compile that way :-(

Reply:
ok